CSVSource/src/main/java/io/vantiq/extsrc/CSVSource/CSVMain.java

```diff
@@ -41,6 +41,8 @@ public class CSVMain {
     static final int NO_AUTH_EXIT = 1;
     static final int NO_SOURCE_EXIT = 2;
     static final int NO_SERVER_EXIT = 3;
+    
```

Comment:
this is a common pattern.  It's seen in many of the connectors that used the security manager